### PR TITLE
LCAM-1483|Disable public URL

### PR DESCRIPTION
**Pull request by venkat980**
## What

[Link to story](https://dsdmoj.atlassian.net/browse/LCAM-1483)

To allow service-to-service communication we have enabled the http port on the mod services. To limit the access to the public url we need to disable the public url and only allow access between services. 

This change is to restrict the public URL in PROD environment. We will still have access in non-prod to public urls to allow testing via postman.

## Checklist

Before you ask people to review this PR:

- [ ] Tests should be passing: `./gradlew test`
- [ ] Github should not be reporting conflicts; you should have recently run `git rebase main`.
- [ ] Avoid mixing whitespace changes with code changes in the same commit. These make diffs harder to read and conflicts more likely.
- [ ] You should have looked at the diff against main and ensured that nothing unexpected is included in your changes.
- [ ] You should have checked that the commit messages say why the change was made.

## Additional checks

- Don’t forget to [run](https://github.com/ministryofjustice/laa-crimeapps-maat-functional-tests/actions/workflows/ExecuteUiTests.yaml) the MAAT functional test suite after deploying your changes to the DEV or TEST environments to ensure your changes haven’t broken any of the functional tests.

### CODE REVIEW

File: helm_deploy/laa-crime-means-assessment/templates/external-ingress.yaml [hunk 1]
```diff
@@ -1,63 +0,0 @@
...
-        {{- end }}
-      secretName: {{ .secretName }}
-    {{- end }}
-  {{- end }}
-  rules:
-    {{- if .Values.ingress.externalHost }}
-    - host: {{ $.Values.ingress.externalHost.host | quote }}
-      http:
-        paths:
-          - path: {{ $.Values.ingress.externalHost.paths.swaggerPath }}
```
Comment (karlbaker02) [inline helm_deploy/laa-crime-means-assessment/templates/external-ingress.yaml:41]: This needs to remain available according to the task

File: helm_deploy/laa-crime-means-assessment/templates/external-ingress.yaml [hunk 2]
```diff
@@ -1,63 +0,0 @@
...
-            pathType: "Prefix"
-            backend:
-              service:
-                name: {{ $fullName }}
-                port:
-                  number: {{ $svcPort }}
-          - path: {{ $.Values.ingress.externalHost.paths.openApiPath }}
```
Comment (karlbaker02) [inline helm_deploy/laa-crime-means-assessment/templates/external-ingress.yaml:48]: This also needs to remain available according to the task

**Review by karlbaker02 — CHANGES_REQUESTED** [review]
Looks good, but there are a few paths removed from the external ingress that we need to keep

File: helm_deploy/laa-crime-means-assessment/templates/external-ingress.yaml (hunk 1 above)
Comment (venkat980) [reply helm_deploy/laa-crime-means-assessment/templates/external-ingress.yaml:41]: WE are no longer using external ingress.

File: helm_deploy/laa-crime-means-assessment/templates/external-ingress.yaml (hunk 2 above)
Comment (venkat980) [reply helm_deploy/laa-crime-means-assessment/templates/external-ingress.yaml:48]: WE are no longer using external ingress.

**Review by karlbaker02 — APPROVED** [review]
Looks good, thanks for clarifying my comments 💯

**Pull request merged**